docs/resources/sagemaker_app_image_config.md

```diff
@@ -10,7 +9,77 @@ description: |-
 
 Resource Type definition for AWS::SageMaker::AppImageConfig
 
+## Example Usage
+
+To create a SageMaker app image config resource.
+
+Basic usage
+
+```terraform
+locals {
```

Comment:
@vivekthangamuthu 
- The tags should be updated as there are no IAM roles here. You can just use below one to keep it consistent across examples. We an ignore the local reference as there is only one resource.
```
    {
      key   = "Modified By"
      value = "AWSCC"
    }
```
Would need to regenerate docs and squash those commits once.

Reply:
@quixoticmonk Thanks for your review, changes has been done now.